### ISSUE TITLE
Tiến độ tiến hành chương Natural Language Processing: Applications

### ISSUE THREAD

**Issue by duythanhvn** [aivivn/d2l-vn#2395]
Cập nhật tiến độ hoàn thành chương **Natural Language Processing: Applications**.
Tiến độ toàn dự án xem **[tại đây](https://github.com/aivivn/d2l-vn/issues/1497)**.

## Giai đoạn 1 `26 parts`
* [x] 15.00 [NLP Index](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+index_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+nlp-application%22) `01 part`
* [x] 15.01. [Sentiment Analysis and the Dataset](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+sentiment-analysis-and-dataset_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+nlp-application%22) `02 parts`
* [x] 15.02. [Sentiment Analysis: Using Recurrent Neural Networks](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+sentiment-analysis-rnn_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+nlp-application%22) `03 parts`
* [x] 15.03. [Sentiment Analysis: Using Convolutional Neural Networks](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+sentiment-analysis-cnn_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+nlp-application%22) `05 parts`
* [x] 15.04. [Natural Language Inference and the Dataset](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+natural-language-inference-and-dataset_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+nlp-application%22) `03 parts`
* [x] 15.05. [Natural Language Inference: Using Attention](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+natural-language-inference-attention_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+nlp-application%22) `05 parts`
* [x] 15.06. [Fine-Tuning BERT for Sequence-Level and Token-Level Applications](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+finetuning-bert_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+nlp-application%22) `04 parts`
* [x] 15.07. [Natural Language Inference: Fine-Tuning BERT](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+natural-language-inference-bert_vn+label%3A%22status%3A+phase+1%22+label%3A%22chapter%3A+nlp-application%22) `03 parts`

## Giai đoạn 2 `11 parts`
* [x] 15.00 [NLP Index](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+index_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+nlp-application%22) `01 part`
* [x] 15.01. [Sentiment Analysis and the Dataset](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+sentiment-analysis-and-dataset_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+nlp-application%22) `01 part`
* [x] 15.02. [Sentiment Analysis: Using Recurrent Neural Networks](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+sentiment-analysis-rnn_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+nlp-application%22) `01 part`
* [x] 15.03. [Sentiment Analysis: Using Convolutional Neural Networks](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+sentiment-analysis-cnn_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+nlp-application%22) `02 parts`
* [x] 15.04. [Natural Language Inference and the Dataset](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+natural-language-inference-and-dataset_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+nlp-application%22) `01 part`
* [x] 15.05. [Natural Language Inference: Using Attention](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+natural-language-inference-attention_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+nlp-application%22) `02 parts`
* [x] 15.06. [Fine-Tuning BERT for Sequence-Level and Token-Level Applications](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+finetuning-bert_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+nlp-application%22) `02 parts`
* [x] 15.07. [Natural Language Inference: Fine-Tuning BERT](https://github.com/aivivn/d2l-vn/issues?q=is%3Aissue+natural-language-inference-bert_vn+label%3A%22status%3A+phase+2%22+label%3A%22chapter%3A+nlp-application%22) `01 part`
